javascript/modules/client/src/internal/interfaces/client.ts

```diff
@@ -82,6 +89,44 @@ export interface IWithdrawParams {
   tokenAddress?: string;
   reference?: string;
 }
+export interface IPermissionParams {
+  where: string;
+  who: string;
+  permission: string;
+}
+export interface IFreezeParams {
+  where: string;
+  permission: string;
+}
+const Permissions = {
+  UPGRADE_PERMISSION: "UPGRADE_PERMISSION",
+  SET_METADATA_PERMISSION: "SET_METADATA_PERMISSION",
+  EXECUTE_PERMISSION: "EXECUTE_PERMISSION",
+  WITHDRAW_PERMISSION: "WITHDRAW_PERMISSION",
+  SET_SIGNATURE_VALIDATOR_PERMISSION: "SET_SIGNATURE_VALIDATOR_PERMISSION",
+  SET_TRUSTED_FORWARDER_PERMISSION: "SET_TRUSTED_FORWARDER_PERMISSION",
+  ROOT_PERMISSION: "ROOT_PERMISSION",
+  CREATE_VERSION_PERMISSION: "CREATE_VERSION_PERMISSION",
+  MULTIPLY_PERMISSION: "MULTIPLY_PERMISSION",
```

Comment:
Some of these permissions are there just for testing purposes (multiply, do something, etc). Could we group them aside of the real ones?
- Prefix them with something like `__TEST_` + name
- Remove them from the declaration and "add" them right after:  `Permissions.__TEST_XX = "TEST_PERMISSION";

Reply:
Or instead, index them in a separate list for testing, and if the Permission ID does not correspond to any of the known "real" permissions, then try on the testing list. But don't export the testing list